mldsa/src/polyvec.c

```diff
@@ -65,12 +66,50 @@ __contract__(
 
 #endif /* !MLD_USE_NATIVE_NTT_CUSTOM_ORDER */
 }
+#endif /* !MLD_CONFIG_REDUCE_RAM */
 
+MLD_INTERNAL_API
+const mld_polyvecl *mld_polymat_get_row(mld_polymat *mat, unsigned int row)
+{
+#if defined(MLD_CONFIG_REDUCE_RAM)
```

Comment:
Please add a TODO here that this branch still needs a CBMC proof. We should open a corresponding issue.